<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>About</class>
 <widget class="QWidget" name="About">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>464</width>
    <height>307</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="mess_label">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>140</y>
     <width>181</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="Home_Button">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>250</y>
     <width>71</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Home</string>
   </property>
  </widget>
  <widget class="QLabel" name="symbol">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>40</y>
     <width>171</width>
     <height>171</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="OK_Button">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>250</y>
     <width>71</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QLabel" name="main_label">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>30</y>
     <width>181</width>
     <height>74</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei Mono</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="lineWidth">
    <number>0</number>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;UnicornTeam&lt;/p&gt;&lt;p&gt;EMV Reader Qt Version&lt;/p&gt;&lt;p&gt;v0.1&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <zorder>symbol</zorder>
  <zorder>mess_label</zorder>
  <zorder>Home_Button</zorder>
  <zorder>OK_Button</zorder>
  <zorder>main_label</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
